BUILD.md

```diff
@@ -284,7 +283,7 @@ General requirements (see [#1469](https://github.com/neovim/neovim/issues/1469#i
 
 - Clang or GCC version 4.9+
 - CMake version 3.16+, built with TLS/SSL support
-  - Optional: Get the latest CMake from an [installer](https://github.com/Kitware/CMake/releases) or the [Python package](https://pypi.org/project/cmake/) (`pip install cmake`)
+  - Optional: Get the latest CMake from https://cmake.org/download/.
```

Comment:
```suggestion
  - Optional: Get the latest CMake from https://cmake.org/download/
    - Provides a shell script which works on most Linux systems. After running it, ensure the resulting `cmake` binary is in your $PATH so the the Nvim build will find it.
```

Also not sure if cmake is smart enough to keep looking in $PATH if it finds an outdated `cmake` before finding the newer one. If not, we should mention that the downloaded `cmake` should be the first item in $PATH.